src/current_user.ts

```diff
@@ -502,6 +528,10 @@ export default class CurrentUser {
       },
       path: `/rooms/${room.id}?message_limit=${messageLimit}`,
     });
+
+    if (roomDelegate.cursorSet) {
```

Comment:
I think this is fine. My only thought was about whether or not it's feasible that someone would only want to set the `cursorSet` property once the subscription to the room had already been established (or at least initiated)?

Reply:
mmm that's a good point. How about a public `subscribeToCursors` method that takes a room ID (and callbacks)?

Reply:
Hmmm - I think what you've got is a really neat solution so I'm hesitant to drop it, even if it does feel a bit magic. The way the presence stuff works is that the subscription is always started and the delegate only gets called if the presence-related functions are implemented. That has its own drawbacks.

Maybe the middle ground, for this and for presence, is to always subscribe by default but allow an option to be provided at the appropriate point (`connect` for presence, `subscribeToRoom` for cursors), which, if set to `false`, won't establish the subscription.

Reply:
Is the advantage of that over what I'm currently doing that you would be able to subscribe initially without providing a callback, and then set the callback at some later date? Either way I like the idea of making the presence and cursors subscription consistent with one another, I should have looked at how presence worked when I was writing this!

Reply:
Basically, yeah. The cost of having an extra subscription running, even if no delegate property to handle the events is set, is approaching nonexistent so I think we should lean on the side of "subscribe by default", especially with multiplexed websockets (or HTTP/2 for other platforms).